pkg/osx/req.txt

```diff
@@ -31,5 +31,4 @@ setproctitle
 singledispatch==3.4.0.3
 smmap==0.9.0
 timelib==0.2.4
-tornado==4.5.3
```

Comment:
You need to add the tornado requirements here too.
I don't believe either the osx or windows packages look at base.txt

Reply:
The tornado requirements are already in this file for some reason.

Reply:
Oh?!
Sorry, Should have checked.